lib/fog/aws/compute.rb

```diff
@@ -551,6 +551,8 @@ def initialize(options={})
           @instrumentor           = options[:instrumentor]
           @instrumentor_name      = options[:instrumentor_name] || 'fog.aws.compute'
           @version                = options[:version]     ||  '2016-11-15'
+          @retry_if_over_limit    = options.has_key?(:retry_if_over_limit) ? options[:retry_if_over_limit] : true
```

Comment:
I wonder if a clearer name might be `retry_request_limit_exceeded`? What do you think?

Reply:
Yup, happy with `retry_request_limit_exceeded` - got hit by: https://martinfowler.com/bliki/TwoHardThings.html